M7/7.ICE.1/src/App.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from 'react'
+import 'regenerator-runtime/runtime' // for using async/await
+
+function App () {
+  const [items, setItems] = useState([])
+  useEffect(() => {
+    getData()
+  }, [])
+
+  async function getData () {
+    const requestURL =
+      'https://raw.githubusercontent.com/rocketacademy/bootcamp-docs/master/fixture-data/products.json'
```

Comment:
We dont need to load from the internet, if you read the instructions they say copy into your index.json